Docs/Generali/glossario/01_content.tex

```diff
@@ -230,6 +230,10 @@ \section{P}
     Nella grafica 3D, movimento della telecamera o della vista che consente 
     di spostarsi orizzontalmente o verticalmente in una scena senza modificarne 
     la prospettiva o l'orientamento.
+    \subsection{PB}
+    \label{PB}
+    (s. p. f.)\\
+    Acronimo di "Product Baseline", indica la seconda e ultima revisione di avanzamento del progetto effettuata dai docenti (committenti) durante la quale si verificano gli obiettivi funzionali e qualitativi del prodotto. La revisione si articola in due parti: al termine della prima, il prodotto viene presentato al proponente per l'accetazione come Minimum Viable Product (MVP).
```

Comment:
errore di battitura: accettazione 
In più, se scrivi due parti devi descriverle entrambe (non solo la prima)

Reply:
Non è una descrizione della prima parte. Prima c'è la revisione di Cardin , poi la presentazione del MVP al proponente e alla fine la seconda revisione con Tullio. 

Adesso cambio con quella delle slide